lib/utils/Commander.ts

```diff
@@ -0,0 +1,227 @@
+import { list } from "@ioredis/commands";
+import asCallback from "standard-as-callback";
+import {
+  executeWithAutoPipelining,
+  shouldUseAutoPipelining,
+} from "../autoPipelining";
+import Command, { ArgumentType } from "../Command";
+import Script from "../Script";
+import { Callback, WriteableStream } from "../types";
+import RedisCommander, { ClientContext } from "./RedisCommander";
+
+export interface CommanderOptions {
+  keyPrefix?: string;
+  showFriendlyErrorStack?: boolean;
+}
+
+const DROP_BUFFER_SUPPORT_ERROR =
+  "*Buffer methods are not available " +
+  'because "dropBufferSupport" option is enabled.' +
+  "Refer to https://github.com/luin/ioredis/wiki/Improve-Performance for more details.";
```

Comment:
That page doesn't exist anymore

Reply:
Good catch! The content was: https://github.com/luin/ioredis/wiki/_compare/c3aeabf0594e0c68b9c5ddf39d8cd7fa4876604a. Going to remove the `dropBufferSupport` option as hiredis is not used anymore.